vocabularies/Common.xml

```diff
@@ -657,7 +658,7 @@ The minimum structure is
         </Annotation>
       </TypeDefinition>
 
-      <Term Name="ValueListRelevantQualifiers" Type="Collection(Common.SimpleIdentifier)">
+      <Term Name="ValueListRelevantQualifiers" Type="Collection(Common.SimpleIdentifier)" Nullable="false">
```

Comment:
Don't I want to use null to address the unqualified value list annotation?

Reply:
We do not specify this yet: could be `null`, could be the empty string. Given that some backends have problems representing `null`, the empty string might be a better choice.

@BerSie: what is Fiori expecting?

Reply:
This knows @uhlmannm better - it's evaluated in the OData model. I pesonally would expect the empty string.

Reply:
@ThomasChadzelek: can you help with this?

Reply:
I checked with Uwe. The value null would not make the default qualifier relevant. Nullable=false seems Ok then.